tools/unix/generate_planet.sh

```diff
@@ -295,6 +297,25 @@ if [ ! -f "$OPENTABLE_FILE" -a -n "${OPENTABLE_USER-}" -a -n "${OPENTABLE_PASS-}
   ) &
 fi
 
+# Download viator.com cities. This takes around 3 seconds.
+if [ ! -f "$VIATOR_FILE" -a -n "${VIATOR_KEY-}" ]; then
+  log "STATUS" "Step S3: Starting background viator cities downloading"
+  (
+      $PYTHON $VIATOR_SCRIPT --apikey $VIATOR_KEY --output "$VIATOR_FILE" 2>"$LOG_PATH"/viator.log || true
+      if [ -f "$VIATOR_FILE" -a "$(wc -l < "$VIATOR_FILE" || echo 0)" -gt 100 ]; then
+        echo "Viator cities have been downloaded. Please ensure this line is before Step 4." >> "$PLANET_LOG"
+      else
+        if [ -n "${OLD_INTDIR-}" -a -f "${OLD_INTDIR-}/$(basename "$VIATOR_FILE")" ]; then
```

Comment:
The variables in the line are processed before parsing it, so all variables must be screened this way.